aio/deploy/helm-chart/kubernetes-dashboard/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: metrics-server
+    version: 2.x.x
+    repository: https://kubernetes-charts.storage.googleapis.com/
+    condition: metricsServer.enabled
```

Comment:
This condition doesn't match the values file or the prefix for other configuration options.  Seems like either the dependency should have an alias or the condition should be changed to `metrics-server.enabled`.

Reply:
Indeed, thanks, fixed!